src/Twig/CodeSelectorTrait.php

```diff
@@ -13,40 +13,13 @@
 
 namespace PhpDocumentGenerator\Twig;
 
-use PhpDocumentGenerator\Parser\Ast\Node;
-use PhpDocumentGenerator\Parser\ClassParser;
-use PhpDocumentGenerator\Parser\ParserInterface;
-use PhpDocumentGenerator\Parser\TypeParser;
-use Twig\TwigFilter;
-
-class MarkdownExtendedExtension extends MarkdownExtension
+// TODO: we could actually do this in twig using a CodeExampleView
```

Comment:
IMHO a trait is good as CodeSelector could be in any element documentation: class, method, etc.